grammars/jsdoc.cson

```diff
@@ -411,6 +413,92 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Strings
+      {
+        'begin': '("|\')'
+        'beginCaptures':
+          '0':
+            'name': 'punctuation.definition.string.begin.js'
+        'end': '\\1|(?=\\*/)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.definition.string.end.js'
+        'name': 'string.quoted.js'
```

Comment:
I'm fine with the rule being duplicated.  Just like how it already is in the main javascript grammar (https://github.com/atom/language-javascript/blob/f470e132e2988d6d088a4746b7a494ed78e3e06c/grammars/javascript.cson#L1338-L1375).